src/store/backend-plugin.ts

```diff
@@ -114,32 +114,54 @@ function _preparePipeline(pipeline: Pipeline, store: Store<any>) {
  *   - logs the error in the store if any,
  *   - sets the `loading: false` property on the store at the end.
  */
-export function backendify(target: Function) {
+export function backendify(
+  target: Function,
+  callFront?: Function,
+  callBack?: Function,
+  reject?: Function,
+) {
```

Comment:
Two comments here:

- I would name those parameters `before`, `after`, `onError` or `before`, `onSuccess`, `onError`

But then… what does `backendify` become useful for? I mean if you specify those 3 parameters, it becomes more or less equivalent to:

```typescript
before()
try {
   target();
} cacth(err) {
   onError(err);
}
after();
```

but just (very) less clear. Am I missing something?

Reply:
Hi tanks Adrien! This can be decided later... Thus I removed the modification on `backendify` from this PR